docs/pages/_app.js

```diff
@@ -311,11 +332,15 @@ function AppWrapper(props) {
         ))}
       </NextHead>
       <ReduxProvider store={redux}>
-        <PageContext.Provider value={{ activePage, pages, versions: pageProps.versions }}>
-          <StylesProvider jss={jss}>
-            <ThemeProvider>{children}</ThemeProvider>
-          </StylesProvider>
-        </PageContext.Provider>
+        <RtlContext.Provider value={rtlContextValue}>
+          <PageContext.Provider value={{ activePage, pages, versions: pageProps.versions }}>
+            <CacheProvider value={rtl ? cacheRtl : cacheLtr}>
```

Comment:
Rtl support for  `emotion` is implemented. Should we do the same for `styled-components`?

Reply:
We have a few customizations demo that uses styled-components now, but I assume we will migrate them all to emotion or even our styled wrapper (for consistency).
So I guess the only use case for configuring rtl would be to help us test and debug issues with styled-components in development. Why not. Sounds interesting.